docs/setup.md

```diff
@@ -0,0 +1,9 @@
+Web Setup Instructions
+-------
+Install [Rails.](http://installrails.com/)
+1.  Clone the Github repo `git clone https://github.com/calblueprint/lavamae.git`
```

Comment:
Make each of these numbered lines appear as a new line - all grouped into one paragraph as of now.